ImpeccableHQ/swapr-dapp
# Linked sdk pr for refferer which now has 0.1% fee for testing purposes

https://github.com/ImpeccableHQ/swapr-sdk/pull/281
- Added 0.1 percent fee for testing purposes I made 2 transaction to confirm its working but would be good to test it out
here are my 2 transactions 
- Gnosis 1inch https://gnosisscan.io/tx/0xd00ac505b2c641e170781158141c21f5d41d6bdd7841f5dfcff504b3309484ff
- Optimism 0xTrade https://optimistic.etherscan.io/tx/0x58d53a4e326cd7f535beaf9ae63152f0a0c3b54c1cab2402abab11870d8e8403


To test out go to transaction:
1) Make a swap with either 1inch or 0xTrade
2) Go to transaction and see if one of the transfers is made to the fee recipient 
<img width="728" alt="Screenshot 2023-02-28 at 11 10 54" src="https://user-images.githubusercontent.com/33226956/221942399-0a19db9e-b62b-4ef9-a73f-235ebd3a161e.png">

3) To verify the that fee reciever is accurate go to this file in the sdk pr on the appropriate chain 
[src/entities/trades/constants/index.ts](https://github.com/ImpeccableHQ/swapr-sdk/pull/280/files#diff-e13660e83886de874808f8e92e58ef2a14b62987b37d044948d1a1cc9dfdde5a)
@0xVenky Provided this list
<img width="677" alt="Screenshot 2023-02-28 at 11 08 30" src="https://user-images.githubusercontent.com/33226956/221941884-1b800a38-e32d-4acf-92d8-ca3d293b467b.png">
